static/app/components/prevent/summary.tsx

```diff
@@ -34,92 +47,116 @@ export function useCreateSummaryFilterLink(filterBy: SummaryFilterKey) {
-  font-size: 2.25rem;
-
-  /* This stops the text from jumping when becoming bold */
-  &::after {
-    content: attr(data-text);
-    height: 0;
-    visibility: hidden;
-    overflow: hidden;
-    pointer-events: none;
-    font-weight: ${p => p.theme.fontWeight.bold};
-    display: block;
-  }
-
-  &[data-is-filtered='true'] {
-    font-weight: ${p => p.theme.fontWeight.bold};
-  }
-
-  &:hover {
-    text-decoration: underline;
-  }
-`;
-
-type SummaryEntryValueLinkProps = {
-  children: React.ReactNode;
-  filterBy: SummaryFilterKey;
-};
+interface SummaryCardProps {
+  label: string;
+  tooltip: ReactNode;
+  extra?: ReactNode;
+  filterBy?: SummaryFilterKey;
+  value?: string | number;
+}
 
-export function SummaryEntryValueLink({children, filterBy}: SummaryEntryValueLinkProps) {
+export function SummaryCard({label, tooltip, value, filterBy, extra}: SummaryCardProps) {
   const {filterLink, isFiltered} = useCreateSummaryFilterLink(filterBy);
 
+  const content = (
+    <Fragment>
+      <Flex align="center" gap="xs">
+        <Heading as="h4" size="sm">
+          {label}
+        </Heading>
+        <QuestionTooltip title={tooltip} size="xs" />
+      </Flex>
+      <Flex justify="between">
+        <Flex align="center" gap="sm">
+          <Text size="xl" bold>
+            {value === undefined ? '-' : value}
+          </Text>
+          {extra}
+        </Flex>
+        {filterBy && (
+          <Button
+            size="zero"
+            borderless
+            title={isFiltered ? t('Clear filter') : t('Filter the table to these tests')}
+            icon={isFiltered ? <IconClose /> : <IconFilter />}
+            aria-label={isFiltered ? t('Clear filter') : t('Filter')}
+          />
+        )}
+      </Flex>
+    </Fragment>
+  );
+
   return (
-    <div style={{display: 'flex', alignItems: 'center', gap: space(0.5)}}>
-      <IconFilter />
-      <Link to={filterLink}>
-        <StyledSummaryEntryValueLink data-is-filtered={isFiltered}>
-          {children}
-        </StyledSummaryEntryValueLink>
-      </Link>
-    </div>
+    <SummaryCardContainer
+      direction="column"
+      padding="md"
+      gap="sm"
+      isFiltered={isFiltered}
+      isClickable={!!filterBy}
+    >
+      {props =>
+        filterBy ? (
+          <Link to={filterLink} {...props}>
+            {content}
+          </Link>
+        ) : (
+          <div {...props}> {content}</div>
+        )
+      }
+    </SummaryCardContainer>
   );
 }
 
-export const SummaryEntry = styled('div')<{columns?: number}>`
-  display: flex;
-  flex-direction: column;
-  align-items: start;
-  justify-content: space-between;
-  grid-column: span ${p => p.columns ?? 1};
+const SummaryCardContainer = styled(Flex)<{isClickable?: boolean; isFiltered?: boolean}>`
+  border: 1px solid ${p => (p.isFiltered ? p.theme.purple300 : p.theme.border)};
+  border-radius: ${p => p.theme.borderRadius};
+  background: ${p => (p.isFiltered ? p.theme.purple100 : 'transparent')};
```

Comment:
@JonasBa would have been nice to get these right on the component, but maybe too custom?

We do have a card component but it wasn't right for this use case.

Reply:
The border radius is just a string though unfortunately. I wish we had tokens there too

Reply:
> The border radius is just a string though unfortunately. I wish we had tokens there too

It should be `radius="md"` for the prop, maybe the props type is bad, I can take a look at that, but iirc it should work